velox/functions/sparksql/specialforms/SparkCastHooks.h

```diff
@@ -23,6 +23,9 @@ namespace facebook::velox::functions::sparksql {
 // This class provides cast hooks following Spark semantics.
 class SparkCastHooks : public exec::CastHooks {
  public:
+  // Returns true.
```

Comment:
Remove the comment.
Maybe you can just implement here.

Reply:
It is to make the difference between Presto and Spark more apparent just like other APIs including legacy() and truncate().
https://github.com/facebookincubator/velox/blob/main/velox/functions/sparksql/specialforms/SparkCastHooks.h#L46-L47

Reply:
I don't know why we use the comment to return the result. I don't think this is a practice example. What do you think? @mbasmanova

Reply:
Updated. Thanks.